mbbsd/register.c

```diff
@@ -1158,6 +1158,18 @@ void
 check_register(void)
 {
     char fn[PATHLEN];
+    int ret = 0;
+
+    // �T�{�@�w�n���p���H�c
+    sprintf(fn, "etc/ensurecontactemail");
+    if(dashf(fn) && !is_valid_email(cuser.email)) {
```

Comment:
This should be determined at compile time, not based on whether some file exists or not, which introduces system call overhead on each login.

This is no longer your old session based BBS. Many apps log in and out rapidly.